README.md

```diff
@@ -14,6 +17,30 @@ JavaScript frameworks.
 It's designed to work with Rails 5.1+ and makes use of the [Yarn](https://yarnpkg.com) dependency management
 that's been made default from that version forward.
 
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Binstubs](#binstubs)
+- [Configuration](#configuration)
+- [Advanced Configuration](#advanced-configuration)
+- [Linking to static assets](#linking-to-static-assets)
+- [Getting asset path](#getting-asset-path)
+- [Deployment](#deployment)
+- [Linking to sprockets assets](#linking-to-sprockets-assets)
+- [Ready for React](#ready-for-react)
+- [Ready for Angular with TypeScript](#ready-for-angular-with-typescript)
+- [Ready for Vue](#ready-for-vue)
+- [Ready for Elm](#ready-for-elm)
+- [Troubleshooting](#troubleshooting)
+- [Wishlist](#wishlist)
```

Comment:
WIshlist? Do you think this should be replaced with "contribution" or simply be removed?